stdlib/test/builtin/test_none.mojo

```diff
@@ -12,11 +12,28 @@
 # ===----------------------------------------------------------------------=== #
 # RUN: %mojo %s
 
+from testing import assert_equal
+
 
 def main():
+    test_str()
+    test_repr()
+    test_format_to()
     test_type_from_none()
 
 
+def test_str():
+    assert_equal(NoneType().__str__(), "None")
+
+
+def test_repr():
+    assert_equal(NoneType().__str__(), "None")
```

Comment:
**Question** Shouldn't this be calling `repr` dunder?

Reply:
Yes it should, good catch and thanks for fixing it for me.
I still have to set up my stdlib development with magic properly.